app/assets/javascripts/student/interventions_controller_templates.hbs

```diff
@@ -0,0 +1,125 @@
+{{#main}}
```

Comment:
The templates weren't available for testing in Jasmine, since they were part of the `show` template.  So this PR pulls them out into a separate file, which involves two kinds of changes.

First, I couldn't find a gem for compiling Mustache templates as part of the asset pipeline, so this PR uses `handlebar_assets`.  Handlebars is a superset of Mustache, but the templates themselves are still just plain Mustache.

Second, it removes the Rails method calls from these templates so that they can be used outside of the Rails view and for Jasmine tests.  They were using the Rails `form_for` functions, so this translates them into plain HTML, since they were static anyway.  And they were also feeding in data about the list of intervention types, and the list of educators.  This now gets rendered as JSON on the page, and then picked up in `student_page.js` like the intervention data.